src/main/java/org/jenkinsci/plugins/workflow/libs/SCMRetriever.java

```diff
@@ -72,7 +74,7 @@ public SCM getScm() {
             return "Legacy SCM";
         }
 
-        public static List<SCMDescriptor<?>> getSCMDescriptors() {
+        public List<SCMDescriptor<?>> getSCMDescriptors() {
```

Comment:
Will not do; this was for internal use only. I will mark it as restricted however.